docs/Advanced-Topics-Custom-Block-Render.md

```diff
@@ -0,0 +1,148 @@
+---
+id: advanced-topics-custom-block-render-map
+title: Custom Block Rendering
+layout: docs
+category: Advanced Topics
+next: advanced-topics-block-components
+permalink: docs/advanced-topics-custom-block-render-map.html
+---
+
+This article discusses how to customize Draft default block rendering.
+The block rendering is used to define supported block types and their respective
+renderers, as well as converting pasted content to known Draft block types.
+
+When pasting content or when using the
+[convertFromHTML](https://facebook.github.io/draft-js/docs/api-reference-data-conversion.html#convertfromhtml)
+Draft will then convert the pasted content to the respective block rendering type
+by matching the Draft block render map with the matched tag.
+
+## Draft default block render map
+
+| HTML element    | Draft block type                        |
+| --------------- | --------------------------------------- |
+| `</h1>`         | header-one                              |
```

Comment:
@mitermayer I assume these are all supposed to be `<hX/>` with a trailing slash, no? :)